rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -324,6 +324,31 @@ private Federation getFederationFromBtcKeys(List<BtcECKey> federationKeys, Insta
         return FederationFactory.buildStandardMultiSigFederation(federationArgs);
     }
 
+    @Test
+    void logCommitFederationFailed() {
+        // arrange
+        Block rskExecutionBlock = arrangeRskExecutionBlock();
+        long executionBlockNumber = rskExecutionBlock.getNumber();
+
+        Federation proposedFederation = P2shErpFederationBuilder.builder().build();
+        byte[] proposedFederationRedeemScriptSerialized = proposedFederation.getRedeemScript().getProgram();
+
+        CallTransaction.Function event = BridgeEvents.COMMIT_FEDERATION_FAILED.getEvent();
+        var topics = new Object[]{};
+        var data = new Object[]{
+            proposedFederationRedeemScriptSerialized,
+            executionBlockNumber
+        };
+
+        // act
+        eventLogger.logCommitFederationFailure(rskExecutionBlock, proposedFederation);
+
+        // assert
+        commonAssertLogs();
+        assertTopics(1);
+        assertEvent(event, topics, data);
+    }
+
     private Block arrangeRskExecutionBlock() {
```

Comment:
How do you feel about moving this to RskTestUtils? Should be useful in many places

Reply:
sure

Reply:
updated